src/cargo/ops/cargo_test.rs

```diff
@@ -158,6 +164,19 @@ fn run_doc_tests(
             .arg("--crate-name")
             .arg(&target.crate_name());
 
+        if doctest_xcompile {
+            p.arg("-Zunstable-options");
+            p.arg("--enable-per-target-ignores");
+        }
+
+        runtool.as_ref().map(|(runtool, runtool_args)| {
+            p.arg("--target").arg(&compilation.target);
```

Comment:
Should this happen unconditionally regardless of whether the runner is set?

Reply:
I think it should be equivalent

Reply:
I don't think that's right though? You can pass `--target` without configuring a runner?

Reply:
Yes you're absolutely correct